src/impl-loader.js

```diff
@@ -12,6 +12,12 @@ function loadImpl(projectRoot) {
   });
 }
 
+function getImplFileList(projectRoot) {
```

Comment:
This may not be the right place for this method. This file is used to load all files into the vm while execution. The static-loader file is used to load files before execution. You could inline this content or extract to another file and use that in other places too.